### ISSUE TITLE
mpegts 本身有支援身份認證的功能嗎

### ISSUE THREAD

**Issue by yanhua1012** [xqq/mpegts.js#194]
請問 mpegts 本身有支援身份認證的功能嗎？例如以 jwt, cookies 、header 的方式帶 token 、帳密的方式。
可以參考文件的哪一個章節，謝謝。

**Comment by xqq** [comment]
@yanhua1012 

> 謝謝您的專業回覆，很具參考加值。我會改至 mpegts.js 討論。
> 另外能否請教，如果以 webSocket 的方式進行連線，
> 上述的 header 與 withCredential 仍有作用嗎？

Nope. The browser will determine whether cookies will be sent or not. We cannot control headers or cookies here.
https://github.com/xqq/mpegts.js/blob/master/src/io/websocket-loader.js#L54

For WebSocket, I'm not familiar with which situation the cookies will or won't be sent. Please investigate them by yourself.
